packages/Babylonian-Core.package/BPExample.class/instance/timeout.st

```diff
@@ -0,0 +1,4 @@
+accessing
+timeout
+
+	^ timeout ifNil: [timeout := 30000]
```

Comment:
breaking coding guidelines to support legacy examples in the image. needed?

Reply:
Very good idea! :)